Jenkinsfile

```diff
@@ -33,6 +34,18 @@ pipeline {
           recordIssues enabledForFailure: true, tool: checkStyle()
           recordIssues enabledForFailure: true, tool: spotBugs()
         }
+        success {
+            stash name: 'binary', includes: 'target/plugin-health-scoring.jar,src/main/docker/Dockerfile'
+        }
+      }
+    }
+
+    stage('Docker image') {
+      /* when {
+          branch 'main' // TODO for now, activated for the PR
+      } */
```

Comment:
will be uncommented before being merged.

Reply:
I would remove it instead:

- There is already a condition for publishing the image in the library: https://github.com/jenkins-infra/pipeline-library/blob/master/vars/buildDockerAndPublishImage.groovy#L205
- It means that you are going to lint + build (+eventually test) the Docker image for each change, which make sense as it's the final artefact (so the application would always be deployable for each commit with successfull check)

WDYT?

Reply:
This is a very good point. I applied your suggestion.